src/types/announcements.ts

```diff
@@ -0,0 +1,14 @@
+// NOTE: Only `New Feature` for now but we can extend this for eg: `Bugfix` or `Rework`
+export type TagVariant = "New Feature"
+
+export type Announcement = {
```

Comment:
are all of the following mandatory or can there be optional keys as well

Reply:
checked w/ nat prior, all the types are mandatory @__@

Reply:
okay cool